frontend/public/components/RBAC/role.jsx

```diff
@@ -249,12 +278,37 @@ export const RolesDetailsPage = (props) => {
         { href: 'bindings', name: t('role~RoleBindings'), component: BindingsForRolePage },
       ]}
       menuActions={menuActions}
+      breadcrumbsFor={
+        props.kind === 'ClusterRole'
+          ? () => getBreadcrumbs('Role', props.kindObj, props.match)
+          : null
+      }
     />
   );
 };
 
 export const ClusterRolesDetailsPage = RolesDetailsPage;
 
+export const RoleBindingsDetailsPage = (props) => {
+  const pages = [navFactory.details(DetailsForKind(props.kind)), navFactory.editYaml()];
+  const actions = [...Kebab.getExtensionsActionsForKind(modelFor(props.kind)), ...common];
+
+  return (
+    <DetailsPage
+      {...props}
+      menuActions={actions}
+      pages={pages}
+      breadcrumbsFor={
+        props.kind === 'ClusterRoleBinding'
+          ? () => getBreadcrumbs('RoleBinding', props.kindObj, props.match)
+          : null
```

Comment:
Again, I'm unsure why the logic is different for a cluster role binding.

Reply:
In this case your are right, I don't need this check since `ClusterRoleBinding` is the only resource mapped to this resource page.  Note: there is no custom `RoleBinding` page, it uses `DefaultDetailsPage`.